handlers/load_test_handler.go

```diff
@@ -39,42 +41,58 @@ func (h *Handler) LoadTestUsingSMPSHandler(w http.ResponseWriter, req *http.Requ
 		http.Error(w, msg, http.StatusInternalServerError)
 		return
 	}
-	benchMark := &models.PerformanceSpec{}
-	if err := yaml.Unmarshal(body, benchMark); err != nil {
+	jsonBody, err := yamlj.YAMLToJSON(body)
+	if err != nil {
+		msg := "unable to convert YAML to JSON"
+		err = errors.Wrapf(err, msg)
+		logrus.Error(err)
+		http.Error(w, msg, http.StatusInternalServerError)
+		return
+	}
+	benchMark := &SMPS.PerformanceTestConfig{}
```

Comment:
Perhaps, “perfTest” or something similar, so that we might be rid of “benchmark” references?

The logic behind being rid of benchmark references is that Meshery’s performance management capabilities are broader in nature than the specific use case of a benchmark.

Reply:
Done